tribe-ext-event-progress-bar.php

```diff
@@ -142,22 +105,40 @@ public function progressbar( $file, $name, $template ) {
 			?>
 
 			<div class="progress-bar-container progress-bar-container__on tribe-common-b2">
-				<div class="progress-bar-container__live" data-start-date="<?php echo $start_date ?>" data-end-date="<?php echo $end_date ?>">
-					<?php echo esc_html_x( 'Live now', 'Label of live event before the progress bar.', 'tribe-ext-event-progress-bar' ); ?>
+				<div class="progress-bar-container__live" data-start-date="<?php esc_attr_e( $start_date ) ?>" data-end-date="<?php esc_attr_e( $end_date ) ?>">
+					<span class="progress-bar-container__live-text">
+						<?php
+						echo esc_html_x( 'Live now', 'Label before the progress bar when event is live.', 'tribe-ext-event-progress-bar' );
+						?>
+					</span>
+					<span class="progress-bar-container__live-text--over tribe-common-a11y-hidden">
```

Comment:
Personal preference - I'd combine this one with the one at the end. One less string to translate and one less element to manage. Just hide the first and leave "Event is over" at the end?

Also, we can use `tribe_get_event_label_singular` for the word "Event" - then it changes if they change the term in the admin.

Reply:
TBH - I could see a case for creating filters for each of these strings. That would allow the user to change them to things like "online now" for virtual events, etc?

Reply:
Did the first reco, it makes sense.
The second (filters) I'm going to leave for now, somehow it doesn't want to work.
Done.